workload/nuo/nuodb-te.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: te
+  labels:
+    app: nuodb
+    group: nuodb
+    deploymentconfig: te
+    type: workload
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: te
+        deploymentconfig: te
+        database: nuodb
+        nodetype: te
+        type: workload
+    spec:
+      containers:
+      -
+        name: te
+        image: nuodb/nuodb-ce:latest
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: NAMESPACE
```

Comment:
@inyee786  namespace format is differnt here , can we make namespace field  like other apps !

Reply:
I have not given the namespace